### ISSUE TITLE
User should see indication that they have tapped

### ISSUE THREAD

**Issue by holmesworcester** [TryQuiet/quiet#1495]
Right now there are no tapped states on iOS or Android for channels and menu items (we used to have them on Android for channels) so users can't tell when they've tapped the screen. 

I think we should be using TouchableOpacity on channels and menu items in action sheets (like settings, delete, etc.)  And TouchableHighlight on buttons. See: https://reactnative.dev/docs/handling-touches

**Comment by EmiM** [comment]
https://github.com/TryQuiet/quiet/pull/1584

**Comment by kingalg** [comment]
1.4.0
Mobile

Implemented correctly.

**Comment by holmesworcester** [comment]
@kingalg @EmiM this isn't done quite right. Two issues:

1. Opacity shouldn't change on drag/scroll since this is non-standard.
2. Opacity change should only last as long as the tap.

GPT4 has some suggestions. For not changing on scroll, it suggests:


```
<TouchableOpacity delayPressIn={150}>
  <Text>Touch Here</Text>
</TouchableOpacity>
```

For the other, when prodded, it suggests using the Pressable API.

https://chat.openai.com/share/92d3c71d-be76-4821-8b98-9c73e6312ebf

**Comment by kingalg** [comment]
@holmesworcester I agree that it's going to look better without change on scroll but I also don't think that it's wrongly implemented by @EmiM . Yesterday when I was checking this issue I was actually missing details of how it should work and I asked Emi how she understood those requirements. It's never a good sign if developer and QA need to guess so we should definitely from the beginning ask for detailed requirements. Opacity not changing on drag/scroll is more common solution but not the only one. To not look far, opacity change on drag/scroll is implemented on reactnative webside that was linked as an example of handling touches. So it was justified that without specific description Emi has chosen this one. What I would take from this is:
1. Let's change how we handle opacity to what Holmes described in the comment - I suggest tracking this in separate issue as this is out of scope of description of this task and it will be easier to check on this later in separate issue.
2. More detail in the task description is better then less even if we think that something is obvious. 
3. If we feel that not all details are clear its important to ask for them.

**Comment by holmesworcester** [comment]
We can add some animations to the Figma prototypes to use as a reference, but from googling quickly I think the word 'tap' has enough specificity (as distinct from dragging or swiping) to make this issue clear.

Another way to approach this is to include a short screencast of every UX related feature for feedback, or at least for every animated one (with screenshots as an option for static UX related features.) I think this is already our general policy and it helps a lot.

**Comment by holmesworcester** [comment]
Creating another issue, as per our discussion.

**Comment by holmesworcester** [comment]
I think this issue covers us for future work: https://github.com/TryQuiet/quiet/issues/1063
